### PR TITLE
Add required security context to mongo pod

### LINKED ISSUE #140: Mongo StatefulSet doesn't set correct security context
The Mongo DB StatefulSet pod doesn't set the correct security context parameters. This means that even if you have configured it to use a service account, and have set up the correct PSP, the PSP may not be chosen because the pod doesn't declare that it needs to run as non root, and that it needs `CAP_CHOWN`, `CAP_SETUID` and `CAP_SETGID`.

### PR DESCRIPTION
**Pull request by Niksko**
This pod runs as root, and requires CHOWN, SETUID and SETGID to work
properly. These should be declared on the security context of the
pod/container in order for the correct PSP to be selected at admission
time, if in use.

Signed-off-by: Nikolas Skoufis <nskoufis@seek.com.au>

@Jasstkn 
@rajdas98 
@ispeakc0de 

#### What this PR does / why we need it:
Adds security context parameters required when using a PSP

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #140 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by Jasstkn — APPROVED** [review]
Hi! Thank you for your contribution! 
I don't have any objections regarding that. Approved.

File: charts/litmus-2-0-0-beta/templates/mongo-sts.yaml [hunk 1]
```diff
@@ -27,13 +27,21 @@ spec:
       {{- if .Values.mongo.serviceAccountName }}
       serviceAccountName: {{ .Values.mongo.serviceAccountName }}
       {{- end }}
+      securityContext:
```
Comment (rajdas98) [inline charts/litmus-2-0-0-beta/templates/mongo-sts.yaml:30]: Can you put this in values.yaml?

File: charts/litmus-2-0-0-beta/templates/mongo-sts.yaml [hunk 2]
```diff
@@ -27,13 +27,21 @@ spec:
...
+        runAsNonRoot: false
       containers:
         - name: mongo
           image: {{ .Values.image.imageRegistryName }}/{{ .Values.mongo.image.repository }}:{{ .Values.mongo.image.tag }}
           ports:
             - containerPort: {{ .Values.mongo.containerPort }}
               name: http
           imagePullPolicy: {{ .Values.mongo.image.pullPolicy }}
+          securityContext:
```
Comment (rajdas98) [inline charts/litmus-2-0-0-beta/templates/mongo-sts.yaml:39]: Same here

**Review by rajdas98 — CHANGES_REQUESTED** [review]

**Comment by Jasstkn** [comment]
Hey. I'm going to close that one because we added support for bitnami mongodb or you can use your own mongo instance and configure it properly according to the requirements.

**Pull request closed without merging**